.travis.yml

```diff
@@ -1,25 +1,29 @@
 language: php
 
-php:
-  - 7.1
-  - 7.0
-  - 5.6
-  - 5.5
-  - 5.4
-  - 5.3
-  - nightly
-  - hhvm
-  - hhvm-nightly
+dist: trusty
+sudo: false
 
 matrix:
+  include:
+    - php: 5.3
+      dist: precise
+    - php: 5.4
+    - php: 5.5
+    - php: 5.6
+    - php: 7.0
+    - php: 7.1
+    - php: nightly
+    - php: hhvm
+    - php: hhvm-nightly
   fast_finish: true
   allow_failures:
     - php: nightly
     - php: hhvm-nightly
 
 install:
-  - composer install
+  - composer install --prefer-dist
```

Comment:
You'll want to prefer source here https://github.com/aidantwoods/SecureParsedown/commit/36936417923f3033b68bef6194902ebd093f7fa4

Reply:
Aaaaaah, and I was wondering why the files are missing :laughing: Thanks!